8Knot/pages/affiliation/visualizations/commit_domains.py

```diff
@@ -66,45 +124,42 @@
                                             step=1,
                                             value=10,
                                             size="sm",
+                                            className="dark-input",
+                                            style={"width": "80px"},
                                         ),
                                     ],
-                                    className="me-2",
-                                    width=2,
-                                ),
-                            ],
-                            align="center",
-                        ),
-                        dbc.Row(
-                            [
-                                dbc.Col(
-                                    dcc.DatePickerRange(
-                                        id=f"date-picker-range-{PAGE}-{VIZ_ID}",
-                                        min_date_allowed=dt.date(2005, 1, 1),
-                                        max_date_allowed=dt.date.today(),
-                                        initial_visible_month=dt.date(dt.date.today().year, 1, 1),
-                                        clearable=True,
-                                    ),
                                     width="auto",
+                                    className="me-4",
                                 ),
                                 dbc.Col(
-                                    dbc.Button(
-                                        "About Graph",
-                                        id=f"popover-target-{PAGE}-{VIZ_ID}",
-                                        color="secondary",
-                                        size="sm",
-                                    ),
+                                    [
+                                        # dbc.Label(
+                                        #     "Date Range:",
+                                        #     style={"marginBottom": "8px", "fontSize": "14px"}
```

Comment:
Sorry if this feels too nitpicky but was this comment just a result of experimentation during development? or is it something else (like a part of the UI that needs a new home in the rewrite)? I'd recommend either acting on it or removing it